lib/puppet/pops/binder/lookup.rb

```diff
@@ -137,7 +137,14 @@ def self.search_for(scope, type, name, options)
    end
   end
 
-  # This is the method called from the puppet/parser/functions/lookup.rb
```

Comment:
Please make it private, and also specific to only do the hiera call (the Lookup does not need a general way to call any function). Other helper methods should also be private if they are not.